superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/FilterScope/FilterScope.tsx

```diff
@@ -98,49 +83,20 @@ const FilterScope: FC<FilterScopeProps> = ({
   useEffect(() => {
     const updatedFormValues = {
       scope: initialFilterScope,
-      scoping: initialScopingType,
     };
     updateScopes(updatedFormValues);
-  }, [initialFilterScope, initialScopingType, updateScopes]);
+  }, [initialFilterScope, updateScopes]);
 
   return (
     <Wrapper>
-      <CleanFormItem
-        name={[...pathToFormValue, 'scoping']}
-        initialValue={initialScopingType}
-      >
-        <Radio.Group
-          onChange={({ target: { value } }) => {
-            const scope =
-              value === ScopingType.All
-                ? getDefaultScopeValue(chartId)
-                : lastSpecificScope.current;
-            updateFormValues({ scope });
-            setHasScopeBeenModified(true);
-            forceUpdate();
-          }}
-        >
-          <Radio value={ScopingType.All}>{t('Apply to all panels')}</Radio>
-          <Radio value={ScopingType.Specific}>
-            {t('Apply to specific panels')}
-          </Radio>
-        </Radio.Group>
-      </CleanFormItem>
-      <Typography.Text type="secondary">
-        {(formScopingType ?? initialScopingType) === ScopingType.Specific
-          ? t('Only selected panels will be affected by this filter')
```

Comment:
Should we keep this text somewhere? In some sort of tooltip maybe?

Reply:
I think this made more sense when we had the radio group to explain each option. Now that the tree is always visible it seems pretty clear that only selected panels will be affected.

Reply:
Do you agree? If not, I can restore the message and put it at the top.